src/main/scala/scorex/core/consensus/ModifierSemanticValidity.scala

```diff
@@ -1,7 +1,30 @@
 package scorex.core.consensus
 
-sealed trait ModifierSemanticValidity
-case object Absent extends ModifierSemanticValidity
-case object Unknown extends ModifierSemanticValidity
-case object Valid extends ModifierSemanticValidity
-case object Invalid extends ModifierSemanticValidity
\ No newline at end of file
+sealed trait ModifierSemanticValidity {
+  val code: Byte
```

Comment:
This seems ok. But I wonder if it is the best approach. It introduces a dependency between our abstract data types and their low level representation in a block. I think we can accept this PR for now. Maybe add a "TODO" comment reminding ourselves to rethink this in the future. 

(I think that in general it might be a good idea to improve the decoupling of abstract data structures and their low-level representations. But this is something we could do in a future PR.)